cadc-registry/src/main/java/ca/nrc/cadc/reg/CapabilitiesWriter.java

```diff
@@ -95,6 +95,7 @@ public class CapabilitiesWriter {
 
     Namespace vosi = Namespace.getNamespace("vosi", "http://www.ivoa.net/xml/VOSICapabilities/v1.0");
     Namespace vs = Namespace.getNamespace("vs", "http://www.ivoa.net/xml/VODataService/v1.1");
+    Namespace vr = Namespace.getNamespace("vr", "http://www.ivoa.net/xml/VOResource/v1.0");
```

Comment:
XMLConstants has all the namespace URIs so please update to use those (including the other hard-coded ones above).

Reply:
OK.  The `XMLConstants` doesn't have a prefix for all of the known Namespace URIs though.  I've only included the hardcoded ones from `CapabilitiesWriter` and the two that were already there.  Does that match up?